rawdisk/plugins/filesystems/linux.yapsy-plugin

```diff
@@ -0,0 +1,8 @@
+[Core]
+Name = LinuxPlugin
```

Comment:
yes all of most of them, xiafs, ext2, ext3, reiserfs have same id - 83, but it looks like your implementation is for ext2, so i would name plugin ext2.. I'd prefer it to be as focused as possible, one plugin per filesystem